xapian-applications/omega/templates/query

```diff
@@ -49,17 +50,18 @@ document.write("<span title=\""+D+" "+T+"\">Modified:<br><b>"+D+"</b></span>");
 // -->
 </script>
 </head>
+
 <body bgcolor="#ffffff">
 <form name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
 <center>
```

Comment:
To center without using <center> tag we have to add CSS property to its parent, as there is no other tag which center's the content.